leaphymicropython/sensors/tof.py

```diff
@@ -8,6 +8,9 @@
 from leaphymicropython.utils.i2c_address_finder import (  # pylint: disable=import-error
     is_device_address_visible,
 )
+from leaphymicropython.utils.i2c_helper import (  # pylint: disable=import-error
```

Comment:
die import-error moet wel oplosbaar zijn, misschien door in CI de workdir aan het PYTHONPATH toe te voegen 🤔